### DISCUSSION TITLE
How to stop the current prompt?

### DISCUSSION THREAD

**Discussion by 00sapo** [CopilotC-Nvim/CopilotChat.nvim#307] [Q&A] [answered]
Hello,is there a way to stop the remote model from answering the current question?

**Comment by deathbeam** [comment]
Yea you can just use CopilotChatReset or CopilotChat <any_other_question> and it will always cancel the current answer first

**Reply by 00sapo** [reply]
Thanks. however writing a new question from the neovim command line is not comfortable when the question is long...

**Reply by deathbeam** [reply, marked as answer]
Yea I can see that being annoying especially when you dont want to fully reset the chat window. I made PR that should add CopilotChatStop and .stop() method on the api here:

https://github.com/CopilotC-Nvim/CopilotChat.nvim/pull/308

**Reply by 00sapo** [reply]
Thank you so much!

**Reply by zoli** [reply]
I added this to my mappings:
```
 mappings = {
        stop = {
          normal = "<C-c>",
          callback = function()
            local copilot = require "CopilotChat"
            copilot.stop()
          end,
        },
      },
```

So it can get stopped with Ctrl + c shortcut.
